CRM/Member/Form/Membership.php

```diff
@@ -1920,44 +1919,6 @@ public static function processFormContribution(
     //CRM-13981, processing honor contact into soft-credit contribution
     CRM_Contribute_BAO_ContributionSoft::processSoftContribution($params, $contribution);
 
-    if ($contribution) {
```

Comment:
So just to articulate here, the reason for this is because no Contribution Custom fields are exposed on the Back office Membership form right?

Reply:
@seamuslee001 exactly